redhat-developer/odo
# Fixed the problem of not clearing config file after project deletion.

fixes #844 

A new function, DeleteProject() is added to the odo config pkg to clean up the active application and components belonging to the project.

To test : 
1. Create a project
2. Create some apps and components in the same project
3. Delete the project
4. Create a new project with the earlier name and check if the earlier apps are shown or check the .odo config file for their presence.
